pihole-google.txt

```diff
@@ -830,6 +829,465 @@ www.google.tt
+talk.google.com
+talk.l.google.com
+ns2.google.com
+ns3.google.com
+ai.google.com
+dir.google.com
+downloads.google.com
+catalog.google.com
+classroom.google.com
+afp.google.com
+advisor.google.com
+allo.google.com
+america.google.com
+amp.google.com
+answer.google.com
+answers.google.com
+apis.google.com
+plus.l.google.com
+wide-plus.l.google.com
+plus-china.l.google.com
+asia.google.com
+assistant.google.com
+base.google.com
+billing.google.com
+blogger.google.com
+bmt.google.com
+bulletin.google.com
+cast.google.com
+cimsun.google.com
+client1.google.com
+clients.l.google.com
+clients-china.l.google.com
+client2.google.com
+client3.google.com
+client4.google.com
+client5.google.com
+cloud.google.com
+code.google.com
+code.l.google.com
+compare.google.com
+dai.google.com
+drive.google.com
+duo.google.com
+earth.google.com
+elections.google.com
+elements.google.com
+esc.google.com
+europe.google.com
+express.google.com
+ext.google.com
+fiber.google.com
+finance.google.com
+fit.google.com
+flight.google.com
+forms.google.com
+fusion.google.com
+gds.google.com
+get.google.com
+glass.google.com
+goto.google.com
+gradient.google.com
+gw0.google.com
+gw3.google.com
+gw2.google.com
+gwt.google.com
+webtoolkit.l.google.com
+hedwig.google.com
+history.google.com
+history.l.google.com
+hpcase.google.com
+hume.google.com
+ig.google.com
+image.google.com
+ime.google.com
+inbox.google.com
+instant.google.com
+investor.google.com
+ipv6.google.com
+ipv4.google.com
+ipv6.l.google.com
+ipv4.l.google.com
+ish.google.com
+isp.google.com
+jibe.google.com
+jigsaw.google.com
+jump.google.com
+keep.google.com
+kiwi.google.com
+knol.google.com
+knoll.google.com
+kuu.google.com
+labs1.google.com
+lens.google.com
+lp.google.com
+mars.google.com
+material.google.com
+talkgadget.l.google.com
+mini.google.com
+mlt.google.com
+movie.google.com
+mts.google.com
+mts.l.google.com
+mts-china.l.google.com
+nec.google.com
+nexus.google.com
+nol.google.com
+notebook.google.com
+notebook.l.google.com
+ogs.google.com
+on.google.com
+one.google.com
+opt.google.com
+pack.google.com
+peering.google.com
+pixel.google.com
+play.google.com
+plus.google.com
+poly.google.com
+pride.google.com
+print.google.com
+privacy.google.com
+protocol.google.com
+pubserv.google.com
+purchase.google.com
+reader.google.com
+realtime.google.com
+safety.google.com
+sandbox.google.com
+sandbox.l.google.com
+services2.google.com
+sheets.google.com
+slides.google.com
+speech.google.com
+station.google.com
+survey.google.com
+surveys.google.com
+time.google.com
+toolbar.google.com
+toolbox.google.com
+transit.google.com
+travel.google.com
+uol.google.com
+upload.google.com
+large-uploads.l.google.com
+virgin.google.com
+vs.google.com
+voice-search.l.google.com
+wave.google.com
+www.ads.google.com
+www.news.google.com
+www10.google.com
+webdrive-client.l.google.com
+www4.google.com
+www5.google.com
+www5.l.google.com
+www6.google.com
+gfe.core.l.google.com
+www9.google.com
+yp.google.com 							 								
```

Comment:
```suggestion
yp.google.com
```

Reply:
The problem is that there are a few domains with a lot spacings.

Reply:
Done